app/code/Magento/Sales/Model/AdminOrder/Create.php

```diff
@@ -2029,7 +2030,30 @@ protected function _validate()
      */
     protected function _getNewCustomerEmail()
     {
-        return $this->getData('account/email');
+        $email_required = $this->_scopeConfig->getValue(
```

Comment:
Please, avoid underscore in the variable naming

Reply:
As above, this isn’t obeyed by the remaining code and likely require refactor of unrelated code, and so would increase scope of the PR. I’d suggest to pick this up in a separate PR that resolves this for all properties in the class.

Reply:
@driskell it's not a big deal to rename variables in the modern IDE, it requires an additional commit to make code style better.

Reply:
Could it be a problem that it is protected and not private? I worry further change here might delay the PR merge due to unrelated opportunistic change. As it stands I’m happy to help with the other changes but this one feels like it needs a little more thought and a separately considered and reversible changeset so this feature isn’t lost should an issue be raised.